capsules/src/ieee802154/xmac.rs

```diff
@@ -0,0 +1,632 @@
+//! X-MAC protocol layer for low power 802.15.4 reception, intended primarily
+//! to manage an Atmel RF233 radio.
+//!
+//! TODO: Test no-preamble transmission with randomized backoff, requires 3
```

Comment:
Can we move the `TODO`s to non-exposed comments (e.g. just `//`)? I think it reads strange in the docs. If there are specific limitations that a user should be aware of, maybe write those as prose, instead.

Reply:
Moved!